tests/README.rst

```diff
@@ -80,6 +80,153 @@ To do that, run `podman pull quay.io/rhinstaller/anaconda-ci:master` or build
 it locally again.
 
 
+Explanation of the GH configuration
+-----------------------------------
+
+Pull request for master:
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Everything is stored in `.github/workflows/validate.yml` configuration files.
```

Comment:
Let's try that and see. I'm not convinced that it will work because the README is in ``tests`` directory so it has to have ``..`` prefix but maybe this will also work.

Reply:
Right, I think you have to define a proper reference. I just did that in my kickstart-tests PR. [This is the reference](https://github.com/rhinstaller/kickstart-tests/pull/431/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR317) and [this the definition](https://github.com/rhinstaller/kickstart-tests/pull/431/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR376). That [looks reasonable](https://github.com/martinpitt/kickstart-tests/tree/ci-docs#regular-test-runs).